Group Delta debu

ChuyangXiao opened the pull request:
previously, a bug occurs in our system due to the error:

File "/usr/src/app/src/recommendation_system/recommendation_flow/controllers/AlphaController.py", line 24, in get_content_ids
filtered_candidates = AlphaFilter().filter_ids(
File "/usr/src/app/src/recommendation_system/recommendation_flow/filtering/AlphaFilter.py", line 16, in filter_ids
nlp_data = pd.read_pickle('/usr/src/app/src/alpha/df_prompt_labeled.pkl')
File "/usr/local/lib/python3.10/site-packages/pandas/io/pickle.py", line 208, in read_pickle
return pickle.load(handles.handle)
_pickle.UnpicklingError: invalid load key, 'v'.

Consisting of the results from external resources, this issue may be a consequence of the inconsistency between the version of pkl readers in local and external environments. Thus, I used the csv file for the blacklist instead of pkl files in that command. I addition, this time I keep the codes of other groups in my forked repo.

kennethgoodman commented on services/backend/src/recommendation_system/recommendation_flow/controllers/__init__.py:
```diff
@@ -1,15 +1,12 @@
-from src.recommendation_system.recommendation_flow.controllers.CharlieController import (
-    CharlieController,
-)
-from src.recommendation_system.recommendation_flow.controllers.BetaController import (
```
dont delete other teams code

ChuyangXiao replied:
Thank you for reminding me! Since I just noticed that when updating the feed.jsx, retriever.py, and init of controllers, we are modifying codes related to other groups. Thus, I just updated these files to the latest version and added our folder "offline" to the backend as well.

kennethgoodman reviewed (changes requested):
the repo you are making changes from is not up to date, sync first

ChuyangXiao commented:
Thank you for reminding me! Since I just noticed that when updating the feed.jsx, retriever.py, and init of controllers, we are modifying codes related to other groups. Thus, I just updated these files to the latest version and added our folder "offline" to the backend as well.
